test/src/e2e_vm_tests/test_programs/should_pass/language/attributes_abi_name/Forc.toml

```diff
@@ -0,0 +1,9 @@
+[project]
+authors = ["Fuel Labs <contact@fuel.sh>"]
+entry = "lib.sw"
+license = "Apache-2.0"
+name = "attributes_abi_name"
+
+[dependencies]
+std = { path = "../../../../../../../sway-lib-std" }
```

Comment:
We don't need the full-blown `std` here. We can use the reduced `sway-lib-std-core`.